src/disclosures/js/views/question-view.js

```diff
@@ -1,23 +1,64 @@
 'use strict';
 var postVerification = require( '../dispatchers/post-verify' );
 var getFinancial = require( '../dispatchers/get-financial-values' );
+var getSchool = require( '../dispatchers/get-school-values' );
 
 var questionView = {
   $getOptions: $( '.get-options' ),
   $followupNoNotSure: $( '.followup__no-not-sure' ),
   $followupYes: $( '.followup__yes' ),
+  $optionsWrapper: $( '.get-options__dynamic' ),
+  $options: $( '.option' ),
+  $optionsSidebar: $( '.get-options_sidebar' ),
+  $transferCredits: $( '.option__transfer-credits' ),
+  $exploreSchools: $( '.option__explore-schools' ),
+  $workWhileStudying: $( '.option__work-while-studying' ),
+  $takeAction: $( '.option__take-action' ),
   $nextSteps: $( '.next-steps' ),
   $feedback: $( '.feedback' ),
 
+  /**
+   * Initiates the object
+   */
   init: function() {
-    this.bigQuestionListener();
+    var settlementStatus =
+      getSchool.values().settlementSchool || false;
+
+    this.bigQuestionListener( settlementStatus );
+    this.displayOptions( settlementStatus );
+  },
+
+  /**
+   * Show the appropriate content in Step 3 for settlement schools
+   * @param {boolean} settlementStatus Flag if this is a settlement school
+   */
+  displayOptions: function( settlementStatus ) {
+    if ( settlementStatus === true ) {
+      questionView.$optionsWrapper.addClass(
+        'get-options__settlement content_main' );
+      questionView.$transferCredits.remove();
+      questionView.$exploreSchools.remove();
+      questionView.$takeAction.remove();
+      questionView.$options.addClass( 'option__settlement' );
+      questionView.$optionsSidebar.show();
+      questionView.$optionsWrapper.removeClass( 'get-options__dynamic' );
+    } else {
+      questionView.$workWhileStudying.remove();
+      questionView.$optionsSidebar.remove();
+    }
   },
 
-  bigQuestionListener: function() {
+  /**
+   * Listener function for the "big question"/"moment of pause" buttons
+   * @param {boolean} settlementStatus Flag if this is a settlement school
```

Comment:
Minor: could be updated to `isSettlementStatus` so that the answer is either true or false, which is a little clearer when reading the code.